source/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ project(Kraken)
 
 # List of directories to search for CMake modules
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake_modules")
+set(CMAKE_INSTALL_PREFIX "/usr/bin/")
```

Comment:
`INSTALL_PREFIX` should be `/usr/local`

Reply:
In this case, it should rather be `/usr/local/bin`

Reply:
I know it is better to use `/usr/local`, but @kinnou02 thinks it is a tricky change because of the software ecosystem around navitia.

Reply:
My point is that INSTALL_PREFIX doesn't end with bin, it should be /usr or /usr/local or /opt. Binaries are then installed in the bin directory.